.github/PROCESS.md

```diff
@@ -26,3 +26,19 @@ This guide tries to be short and does not mention things that are completely aut
    All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
    Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}." or "Closes #{issue_number}.".
    Do not leave original commit messages in the squashed commit body.
+
+## Team Performance Metrics
+
+1. We want to deliver fast and make our deliverables predictable.
+2. To achieve that, we measure the flow of work and estimate tasks.
+3. The flow of work is measured automatically based on GitHub's pull request workflow.
+4. Tasks are measured manually by engineers during Sprint planning.
+5. Task estimation depends on the scope, the complexity of the task, and the possible risks and clarity.
```

Comment:
```suggestion
5. Task estimation depends on the following parameters:
   * Scope
   * Complexity of the task
   * Possible risks
   * Clarity
```

@AlekSi @noisersup please check if this description for 5 looks better.

Reply:
LGTM!

Reply:
I don't think that risks worth a separate point. We always have risks. We can have extra risks due to complexity on unclarity, but other points cover that already

Reply:
I will update the PR